frontend/src/lib/components/password.svelte

```diff
@@ -30,12 +36,16 @@
 	<!-- Put a title and the numpad -->
 	<div class="flex flex-col items-center bg-neutral-700 rounded-lg shadow-lg p-4 z-40">
 		<h1 class="text-2xl font-bold mb-4 text-white">{custom_text}</h1>
+		{#if warnCapsLockOn}
+			<h2 class="text-2xl font-bold mb-4 text-red">Attention, caps lock on!</h2>
```

Comment:
Faut juste fix la couleur mais sinon c'est good 

```suggestion
			<h2 class="text-2xl font-bold mb-4 text-red-600">Attention, caps lock on!</h2>
```